modules/search/instant-search/components/search-app.jsx

```diff
@@ -110,21 +133,22 @@ class SearchApp extends Component {
 				getSearchQuery(),
 				getFilterQuery(),
 				getSortQuery(),
+				getResultFormatQuery(),
 				this.state.response.page_handle
 			);
 	};
 
-	getResults = ( query, filter, sort, pageHandle ) => {
+	getResults = ( query, filter, sort, resultFormat, pageHandle ) => {
 		const requestId = this.state.requestId + 1;
 
-		this.setState( { requestId, isLoading: true }, () => {
+		this.setState( { requestId, isLoading: true, resultFormat }, () => {
```

Comment:
Potential edge case: `resultFormat` can be set to an empty string when no format query value is provided, overriding the initial state value of `RESULT_FORMAT_MINIMAL`.